apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/InputReceiver/InputReceiver.tsx

```diff
@@ -43,6 +45,58 @@ export const InputReceiver = ({
   )
 }
 
+const Loading = () => {
+  const spin = React.useRef(new Animated.Value(0)).current
+
+  React.useEffect(() => {
+    Animated.loop(
+      Animated.timing(spin, {
+        toValue: 1,
+        duration: 1500,
+        easing: Easing.linear,
+        useNativeDriver: true,
+      }),
+    ).start()
+  }, [spin])
+
+  const getRotationStyle = React.useCallback(() => {
+    const rotate = spin.interpolate({
+      inputRange: [0, 1],
+      outputRange: ['0deg', '360deg'],
+    })
+
+    return {
+      transform: [{rotate}],
+    }
+  }, [spin])
```

Comment:
transform into a hook, useSpin, alike.

Reply:
named the hook `useGetRotationStyle` because it's whats being used.

let me know if you have any suggestion or improvement